opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -77,7 +78,11 @@ def __init__(
                 "severity_text": severity_text,
                 "severity_number": severity_number,
                 "body": body,
-                "attributes": attributes,
+                "attributes": BoundedAttributes(
+                    maxlen=None,
+                    attributes=attributes if bool(attributes) else None,
+                    immutable=False,
+                ),
```

Comment:
`maxlen` can't be None since need to respect the attribute limits configuration. Can you take a look at the tracing implementation and do the same here https://github.com/open-telemetry/opentelemetry-python/blob/e9530c5c548d08a6aaa56268d103f9beb00cd002/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L776-L781

Reply:
Updated.